utilities/differential.py

```diff
@@ -59,13 +59,15 @@ def get_theoryAgnostic_axes():
 
     # Note that the helicity axis is defined elsewhere, and must not be added to the list of axes returned here
     axis_ptVgen = hist.axis.Variable(
-        [0., 5., 10., 15., 20., 25., 30., 35., 40., 45., 50.],
+        #[0., 5., 10., 15., 20., 25., 30., 35., 40., 45., 50.],
+        [0., 3., 6., 9.7,12.4,16.,21.4,29.5,60.],
```

Comment:
Here and for yW we should allow for some flexibility to customise these arrays without having to modify the function itself, but it can also be dealt with by a new PR in the future. The unfolding does something already in this respect, but the binning there tends to be more regular, so it is easier to manage.

Reply:
Yes let's leave this for a subsequent PR